rim/static/rim/js/edit.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+    
+
+    var timer;
+    $('#id_serial_no').on('input', function(e) {
+        clearTimeout(timer)
+        timer = setTimeout(check_serial_nums, 300);
+    })
+
+    function check_serial_nums() {
+        $('.button').prop('disabled', false);
+        var existing_serial_nums = [];
+        $('#serial_no_errors').remove()
```

Comment:
Can this be moved to the top of the `success` callback? That way the page won't jump around if there's no changes to the error list.